app/controllers/KairosDbProxyController.java

```diff
@@ -173,6 +187,29 @@ public KairosDbProxyController(
         }
     }
 
+    MetricsQuery addMergeAggregator(final MetricsQuery metricsQuery) {
+        final List<Metric> newMetrics = new ArrayList<>();
+        for (Metric metric : metricsQuery.getMetrics()) {
+            List<Aggregator> newAggregators = new ArrayList<>();
```

Comment:
Nit: the creation of the new `ArrayList` is unnecessary for the `else` branch. Considering deferring creation into the first branch.